src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/NonTransitResults.tsx

```diff
@@ -42,7 +41,7 @@ export const NonTransitResults = ({tripPatterns, onDetailsPressed}: Props) => {
           <Button
             onPress={() => onDetailsPressed(tripPattern, {analyticsMetadata})}
             style={style.tripMode}
-            key={modeText}
+            key={modeText + i}
```

Comment:
Shouldn't use index in React keys. Some explanation in the pitfall explanation here:
https://react.dev/learn/rendering-lists#why-does-react-need-keys

With this fix, it should be good enough to just use `modeText` as before?